### PR TITLE
Migrate to new contract and fetch both withdrawEvents

### LINKED ISSUE #1: Use new contract as default (deprecate old one)
**old contract:**

https://sandgarden.buidlguidl.com/members
https://optimistic.etherscan.io/address/0x2eA63c9C9C114ae85b1027697A906420a23e8572

**new contract:** 

https://sandgarden.buidlguidl.com/members2
https://optimistic.etherscan.io/address/0x964d0C9a421953F95dAF3A5c5406093a3014A5D8

----

Lets remove the old contract from the UI, and members2 => members.

If some ETH are remaining, people can still withdraw from it. e.g. using https://abi.ninja/0x2eA63c9C9C114ae85b1027697A906420a23e8572/optimism

### LINKED ISSUE #2: Fetch events from old + new contract
On the projects page. we want to display the history of both contracts. 

image

`packages/nextjs/generated/hardhat_contracts.ts` had the info for both.
See #1 for more info.

---

If we end up having a [BG ponder indexer](https://ponder.sh/), this will be way more performant ;)

### PR DESCRIPTION
**Pull request by technophile-04**
### Description : 

Renamed old contract (YourContract) => `_SandGardenStreamsOld` and used new contract `SandGardenStreams` 

 Fixes #1 
 
 Also tackled #2, fetching events from both contracts and displaying them since it didn't require a lot of changes, maybe we can close #2 and create a new issue with "Experiment with ponder.sh api"?

### CODE REVIEW

**Comment by technophile-04** [comment]
also added a small loading state [d3eb7c4](https://github.com/BuidlGuidl/sandgarden.buidlguidl.com/pull/4/commits/d3eb7c419fccde1b7242b1bbf79664c14d80f282) previously there was no indicator for that : 

https://github.com/BuidlGuidl/sandgarden.buidlguidl.com/assets/80153681/7c15e896-82dc-4221-a06e-45aeefbc1640

Lol we should try migrating to ponder.sh api, because fetching events from both contracts takes some time

File: packages/nextjs/scaffold.config.ts [hunk 1]
```diff
@@ -26,6 +27,15 @@ const scaffoldConfig = {
   // It's recommended to store it in an env variable:
   // .env.local for local testing, and in the Vercel/system env config for live apps.
   walletConnectProjectId: process.env.NEXT_PUBLIC_WALLET_CONNECT_PROJECT_ID || "3a8170812b534d0ff9d794f19a901d64",
+
+  contracts: {
+    _SandGardenStreamsOld: {
+      fromBlock: 101114639,
```
Comment (technophile-04) [inline packages/nextjs/scaffold.config.ts:33]: Not sure what's the place to add this 😅 just wanted this value to be shared with others cloning the repo

Comment (carletex) [reply packages/nextjs/scaffold.config.ts:33]: This is great! I was using env vars, so you can have different values on different environments/chains... but I think this is more straight-forward.

Made a few tweaks here b43a70c4f7b1f3be61e03a2ff271fa06edd0096b

**Comment by carletex** [comment]
Thank you for this @technophile-04 it looks great!!

>  "Experiment with ponder.sh api"?

I think we should do this in a standalone project and deploy it as its own. Started with. https://github.com/carletex/ponder-bg-indexer

which will index all the contracts that the BG controls... so we can query them easily with GraphQL from here and all the other cohorts / bg.com, etc.

**Pull request merged**